### ISSUE TITLE
send to minecraft 

### ISSUE THREAD

**Issue by invisiblespider** [ImUrX/vscode-scarpet#13]
![send to minecraft](https://user-images.githubusercontent.com/62538833/78670745-e780a380-78e6-11ea-9198-229bf9f0fcc7.png)

instead of reloading and running the code inside minecraft, vs code sends the code directly to minecraft to run so you can quickly iterate over ideas. 
there is a maya extension that does that, it would be a cool addition

**Comment by ImUrX** [comment]
will need to implement kind of a LSP to the carpet mod, will check it out with gnembon

**Comment by Ghoulboy78** [comment]
what is an lsp, and also there is no need for that man. You just save it into scripts folder, then go into game and type `/script load scriptname`. It doesnt get faster than that

**Comment by ImUrX** [comment]
im talking about a debugger you genius

**Comment by ImUrX** [comment]
and an LSP is just an addon but as a server

**Comment by Ghoulboy78** [comment]
yeah but then you have to implement that stuff directly into carpet mod, no real other way is there?

**Comment by ImUrX** [comment]
I never replied by here, you would need some high permissions as an app there is no other "nice" way of doing it

**Comment by Parietic** [comment]
Gonna leave this here incase it helps anyone.

What I'm currently doing to quickly test scripts is using this extension: [Run on Save](https://marketplace.visualstudio.com/items?itemName=emeraldwalk.RunOnSave)
With these settings:
```json
"emeraldwalk.runonsave": {
    "commands": [
        {
            "match":"\\.sc$",
            "cmd":"copy ${file} C:\\Users\\USER\\AppData\\Roaming\\.minecraft\\config\\carpet\\scripts\\${fileBasename}"
        }
    ]
}
```
This lets me automatically copy any script within my workspace, to the scripts directory in MC each time the script is saved.
Combine this with `"files.autoSave": "afterDelay"` and it feels pretty smooth.
All I need to do after I make a modifcation is alt-tab and reload the script.

Will say though, having vs code auto reload the script and then have MCs chat/logs in vs codes output would be amazing. But Sadly doubt we can do that with out carpet providing a LSP.

**Comment by ImUrX** [comment]
kind of possible with carpet extensions but you need some kind of mod in the client and maybe in the server or you can do what's said above lol
